### PR TITLE
Expand `$args` array parameters that are described in other functions

### LINKED ISSUE #46: Expand `$args` array parameters that are described in other functions
Basically the same as https://github.com/WordPress/wporg-developer/pull/96 but for the `@phpstan-param` tags in these stubs.

### PR DESCRIPTION
**Pull request by johnbillion**
Fixes #46. Needs testing.

## Todo

* [ ] Check each of the ~60 additions to ensure they're correct
* [ ] Add inline docs so our future selves are thankful

### CODE REVIEW

**Comment by szepeviktor** [comment]
> `strpos($description, '()')`

Very clever.

**Comment by johnbillion** [comment]
This is looking good and we're green ✅ but I want to manually check each of the additions to ensure their correctness because there is an allowance for the parameter names to differ from its inherited function, which could in theory result in the wrong parameter being used.

File: visitor.php [hunk 1]
```diff
@@ -364,32 +478,140 @@ private function addArrayHashNotation(Doc $docComment): ?Doc
...
+        ];
+        $matchingTags = array_filter($tags, static function(WordPressTag $tag) use ($matchNames): bool {
+            return in_array($tag->name, $matchNames, true);
+        });
+
+        foreach ($matchingTags as $tag) {
+            $addTag = clone $tag;
+            $addTag->name = $paramName;
+            $addTag->description = sprintf(
+                '@see %s()',
```
Comment (szepeviktor) [inline visitor.php:591]: This is the greatest feature of this PR! (no sarcasm)

Please remove the `@` sign as it is a tag-in-a-tag.

Comment (szepeviktor) [reply visitor.php:591]:
```suggestion
                'See %s()',
```

**Comment by szepeviktor** [comment]
Thank you!

**Pull request merged**